src/elements/card/src/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx, useThemeUI } from 'theme-ui'
+
+import { ImgixImage } from '../../imgix-image/src'
+
+interface Props {
+  imgSrc: string
+  imgAlt: string
+  title: string
+  description: string
+  linkHref: string
+  linkText?: string
+}
+
+const Card: React.FC<Props> = ({
+  imgSrc,
+  imgAlt,
+  title,
+  description,
+  linkHref,
+  linkText = 'Read more'
+}) => {
+  const { theme }: any = useThemeUI()
+
+  return (
+    <div
+      sx={{
+        display: 'flex',
+        flexDirection: theme.card.flexDirection ? ['column', 'row'] : 'column',
+        border: theme.card.flexDirection ? 'none' : 'solid 1px #E5E5E5',
+        padding: theme.card.flexDirection ? '0' : 'xs',
+        marginTop: '0',
+        marginRight: 'xs',
+        marginBottom: 'xs',
+        maxWidth: theme.card.flexDirection ? ['245px', '450px'] : '245px'
+      }}
+    >
+      <ImgixImage
```

Comment:
Also I feel like this entire image should be passed in as a prop so that the component isn't bound to imgix

Reply:
I also had the same thought but Jonathan advised to do it this way...